data/remnant missions.txt

```diff
@@ -655,149 +655,288 @@ mission "Remnant: Learn Sign Follow Up"
+			`You remember that Taely was interested in seeing examples of technology from human space. Would you like to show her your heavy laser cannons?`
 			choice
-				`	"They're heavy lasers."`
-				`	"I'd rather not share that."`
-					decline
-			
-			`	"Really? Our records indicate that humanity had primitive laser technology at the time of the Exodus, but nothing that would be considered useful as weapons." She pauses to look down at her scanner. "And yet our scans indicate that they are capable of significant power output.`
-			`	"Based on these scans, we may well be able to learn quite a bit from these weapons. Would you be able to acquire a pair of them for us? We could offer you very good compensation for doing so."`
-			choice
-				`	"Sure, I would be glad to accept that deal."`
-				`	"Sorry, I'm not really interested in helping Remnant research."`
-					decline
-
-			`	After a bit of haggling, she agrees to pay you <payment> in exchange for two heavy lasers.`
+				`	Not now.`
+					defer
+				`	Yes.`
+			branch stuff
+				has "Remnant Tech Retrieval: done"
+			`	After asking a few people, you finally get directed to a bay in the shipyard where you find Taely just finishing the installation of a thrasher cannon in a starling. "Greetings, <first>. I'll be down in a minute." She does a few more things inside a panel, reseals the hatch, and slides down a fin to land next to you. "So, do you have something for me?"`
+				goto the_rest
+			label stuff
+			`	Remembering that she spends most of her time in the shipyard, you head straight there and ask the nearest mechanic. They direct you to a large hanger tucked into a cliff face, where you find her working at a terminal filled with schematics. Behind her, the room fades quickly into darkness where you can faintly make out what appears to be a large tank and a lot of whirring machinery. As you approach she looks up from her work and turns to face you. "Ah, you have returned. Do you have something new for me?"`
+			label the_rest
+			`	You lead her back to your ship, where you show her the heavy laser cannon. "A laser cannon?" she asks. "Our records indicate that humanity had primitive laser technology at the time of the Exodus, but nothing that would be considered useful as weapons." She pauses to look down at her scanner. "And yet my scans indicate that they are capable of significant power output." She pauses for a moment before continuing, "These could be quite useful to examine more thoroughly. Please deliver two of these heavy laser cannons to a research team on <planet>."`
 				accept
 	on complete
 		outfit "Heavy Laser" -2
 		payment 870000
 		"remnant met taely" ++
```

Comment:
This condition can be removed from all these retrieval missions as it is currently set by the mission that precedes them all.

Reply:
If it works correctly, then the first of these missions that the player does (which will be done while the initial mission is still active and thus not done) will get the one option, whereas the rest of the missions (after the player has already done one, and thus completed the Tech Retrieval mission) get the alternate text where they are a bit more familiar with Taely.

Reply:
Just realizing you were probably actually referring to the `"remnant met taely" ++` line, not the branch above it.

While triggering the Taely log entry is currently the only visible use for this variable, I do want to track how many times the player helps Taely in certain forms. There will be missions that trigger events based on this in the future.